modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/ZonePartitionRaftListener.java

```diff
@@ -72,6 +82,26 @@ private static class CommittedConfiguration {
         }
     }
 
+    private final FinishTxCommandHandler finishTxCommandHandler;
+
+    /** Constructor. */
+    public ZonePartitionRaftListener(
+            TxStatePartitionStorage txStatePartitionStorage,
+            TxManager txManager,
+            SafeTimeValuesTracker safeTimeTracker,
+            PendingComparableValuesTracker<Long, Void> storageIndexTracker,
+            ZonePartitionId partitionId
+    ) {
+        this.safeTimeTracker = safeTimeTracker;
+        this.storageIndexTracker = storageIndexTracker;
+
+        finishTxCommandHandler = new FinishTxCommandHandler(
+                txStatePartitionStorage,
+                new TablePartitionId(partitionId.zoneId(), partitionId.partitionId()),
```

Comment:
In what ticket you expect it to be fixed?

Reply:
I thought about https://issues.apache.org/jira/browse/IGNITE-24343

I didn't add a TODO as the compiler will not allow me to leave a `TablePartitionId` here after IGNITE-24343 is fixed